src/dodal/beamlines/i10.py

```diff
@@ -255,3 +257,48 @@ def idd_la_angle(
         wait=wait_for_connection,
         fake=fake_with_ophyd_sim,
     )
+
+
+@device_factory()
+def m1() -> Mirror_with_piezo:
+    return Mirror_with_piezo(prefix="BL10I-OP-COL-01:")
```

Comment:
```suggestion
    return Mirror_with_piezo(prefix=f"{PREFIX.beamline_prefix}-OP-COL-01:")
```

Slightly less relevant for the i10 specific slits, but if we use this standard form of passing the beamline prefix into the PV then we can more easily re-use the devices on other beamlines.

Reply:
n.b. requires defining PREFIX = BeamlinePrefix(BL) at the top of the module ;P